cypress/integration/1225-my_institution/361-institution_administrator_edits_my_institution.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Institution Administrator edits My Institution
+
+    @360
+    Scenario: Institution Administrator opens My Institution
+        Given that the user is logged in as Institution Administrator
+        When they click the menu item My Institution
+        Then they see the My Institution page
+        And they see fields:
+            | Name in organization registry |
+            | Display name                  |
+            | Short display name            |
+            | Archive name                  |
+        And they see the Save button
+
+    @361
+    Scenario: Institution Administrator edits My Institution
+        Given that the user is logged in as Institution Administrator
+        And they click the menu item My Institution
+        And they see the My Institution page
+        When they edit fields:
+            | Name in organization registry |

```

Comment:
denne bør sikkert oppdateres (om den ikke er det allerede), for denne skal vel ikke kunne endres?

Reply:
Skal legge inn en PR på det